### PR TITLE
Update GPG create documentation

### PR DESCRIPTION
**Pull request by dwhenry**
* Added link to next step in the process - encrypt heira data
* Remove flag from example that is obsolete in latest version of GPG

### CODE REVIEW

File: source/manual/create-a-gpg-key.html.md [hunk 1]
```diff
@@ -4,10 +4,12 @@ title: Create a GPG key
 parent: "/manual.html"
 layout: manual_layout
 section: Support
-last_reviewed_on: 2017-04-01
+last_reviewed_on: 2017-09-06
 review_in: 6 months
 ---
 
+We use GPG keys to encryt our sercets, details for getting access to with a GPG key can be found [here](/manual/encrypted-hiera-data.html#common-tasks-for-handling-encrypted-hiera-data).
```
Comment (rubenarakelyan) [inline source/manual/create-a-gpg-key.html.md]: Couple of spelling corrections: "encrypt" and "secrets"

Comment (rubenarakelyan) [reply source/manual/create-a-gpg-key.html.md]: Also, I think a word is missing here: "getting access to ? with a GPG key can be found"

Comment (dwhenry) [reply source/manual/create-a-gpg-key.html.md]: reworded to clarify what the link is for

Comment (dwhenry) [reply source/manual/create-a-gpg-key.html.md]: reworded to clarify what the link is for

**Pull request merged**